pvlib/iotools/pvgis.py

```diff
@@ -0,0 +1,176 @@
+
+For more information, see the following links:
+* `Interactive Tools <https://re.jrc.ec.europa.eu/pvg_tools/en/tools.html>`_
+* `Data downloads <https://ec.europa.eu/jrc/en/PVGIS/downloads/data>`_
+* `User manual docs <https://ec.europa.eu/jrc/en/PVGIS/docs/usermanual>`_
+
+More detailed information about the API for TMY and hourly radiation are here:
+* `TMY <https://ec.europa.eu/jrc/en/PVGIS/tools/tmy>`_
+* `hourly radiation
+  <https://ec.europa.eu/jrc/en/PVGIS/tools/hourly-radiation>`_
+* `daily radiation <https://ec.europa.eu/jrc/en/PVGIS/tools/daily-radiation>`_
+* `monthly radiation
+  <https://ec.europa.eu/jrc/en/PVGIS/tools/monthly-radiation>`_
+"""
+import io
+import requests
+import pandas as pd
+from pvlib.iotools import parse_epw
+
+URL = 'https://re.jrc.ec.europa.eu/api/'
+
+
+def get_pvgis_tmy(lat, lon, outputformat='json', usehorizon=True,
+                  userhorizon=None, startyear=None, endyear=None, url=URL,
+                  timeout=30):
+    """
+    Get TMY data from PVGIS [1]_. For more information see the PVGIS TMY tool
+    documentation [2]_.
+
+    Parameters
+    ----------
+    lat : float
+        Latitude in degrees north
+    lon : float
+        Longitude in dgrees east
+    outputformat : str, default 'json'
+        Must be in ``['csv', 'basic', 'epw', 'json']``. See PVGIS TMY tool
+        documentation [2]_ for more info.
+    usehorizon : bool, default True
+        include effects of horizon
+    userhorizon : list of float, default None
+        optional user specified elevation of horizon in degrees, at equally
+        spaced azimuth clockwise from north, only valid if `usehorizon` is
+        true, if `usehorizon` is true but `userhorizon` is `None` then PVGIS
+        will calculate the horizon [3]_
+    startyear : int, default None
+        first year to calculate TMY
+    endyear : int, default None
+        last year to calculate TMY, must be at least 10 years from first year
+    url : str, default :const:`pvlib.iotools.pvgis.URL`
+        base url of PVGIS API, append ``tmy`` to get TMY endpoint
+    timeout : int, default 30
+        time in seconds to wait for server response before timeout
+
+    Returns
+    -------
+    data : pandas.DataFrame
+        the weather data
+    months_selected : list
+        TMY year for each month, ``None`` for basic and EPW
+    inputs : dict
+        the inputs, ``None`` for basic and EPW
+    meta : list or dict
+        meta data, ``None`` for basic
+
+    Raises
+    ------
+    requests.HTTPError
+        if the request response status is ``HTTP/1.1 400 BAD REQUEST``, then
+        the error message in the response will be raised as an exception,
+        otherwise raise whatever ``HTTP/1.1`` error occurred
+
+    References
+    ----------
+
+    .. [1] `PVGIS <https://ec.europa.eu/jrc/en/pvgis>`_
+    .. [2] `PVGIS TMY tool <https://ec.europa.eu/jrc/en/PVGIS/tools/tmy>`_
```

Comment:
Anyone else looked into why "(1, 2)" appears at the head of this citation?  Looks like a sphinx-RTD bug to me, it also appears on the [sphinx demo](https://github.com/readthedocs/sphinx_rtd_theme/blob/a3ab477aaa23f3b7ab7d62c7abc2cc74102ab2b8/docs/demo/demo.rst#id59)

Only commenting in case someone has looked into it, and for the discussion in #837

Reply:
The same question came up here: https://github.com/pvlib/pvlib-python/pull/833#issuecomment-565499186 -- I think that behavior is intentional for back-linking in cases where a citation is referenced more than once.  Usually you click on the `[1]` in a citation to go to where the citation is referenced in the text, but sphinx links them each separately if there are multiple references.

Reply:
Right, backlinks. Thanks!